release-plz.toml

```diff
@@ -18,3 +20,9 @@ body = """
 {% endfor -%}
 {% endfor -%}
 """
+
+[[package]]
+name = "vortex-array"
+git_tag_enable = true
+git_release_enable = true
+git_tag_name = "v{{ version }}"
```

Comment:
can i say something without people getting mad

![image](https://github.com/user-attachments/assets/2cdc96a0-79a7-405b-a146-66da148bb5ad)

Reply:
Nice, I missed this